src/loaders/workspace_downloader/workspace_downloader.py

```diff
@@ -438,10 +459,12 @@ def main():
         conf = Conf(
```

Comment:
You should use keynames here rather than positional arguments for backwards compatibility in the future

Reply:
Updated!

Reply:
:+1: